core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/bundle/BundleRequests.java

```diff
@@ -18,27 +18,25 @@
 /**
  * @since 8.0
  */
-public final class BundleRequests {
+public class BundleRequests {
 	
-	private BundleRequests() {}
-	
-	public static BundleCreateRequestBuilder prepareNewBundle() {
+	public static BundleCreateRequestBuilder prepareCreate() {
 		return new BundleCreateRequestBuilder();
 	}
 
-	public static BundleGetRequestBuilder prepareGetBundle(final String id) {
+	public static BundleGetRequestBuilder prepareGet(final String id) {
 		return new BundleGetRequestBuilder(id);
 	}
 	
-	public static BundleSearchRequestBuilder prepareSearchBundle() {
+	public static BundleSearchRequestBuilder prepareSearch() {
```

Comment:
These are still static resources and you are referring to them like that. Could you please turn them into member methods and use them like this `ResourceRequests.bundles().prepareSearch()`.